### PR TITLE
update CrudTrait

### PR DESCRIPTION
**Pull request by villers**
optimize syntax file

### CODE REVIEW

**Comment by tabacitu** [comment]
Thank you @villers , it does look better. I'll test it tomorrow and merge it.

Cheers!

**Comment by villers** [comment]
I like this project! I'll do other optimization :)

File: src/CrudTrait.php [hunk 1]
```diff
@@ -56,10 +54,8 @@ public function addFakes($columns = ['extras'])
                 $column_contents = json_decode($this->{$column});
             }
 
-            if (count($column_contents)) {
```
Comment (tabacitu) [inline src/CrudTrait.php]: This didn't work though. It badly needed that "if" :-)

**Comment by tabacitu** [comment]
Great to hear that! Looking forward to them :-)

**Pull request merged**